recipes/docopt.cpp/0.6.2/test_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class DoctestTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def configure(self):
+        if tools.cross_building(self.settings):
+            del self.settings.compiler.libcxx
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if tools.cross_building(self.settings):
```

Comment:
note affecting review
this code block appears to be recurring - maybe we need to extract helper? @danimtb @uilianries

Reply:
It would need to be smarter than what's in the code (as building from win x64 to win x86 is a cross-build, but in this case we want to run the executable)

Reply:
@SSE4 Indeed